apps/cli/lib/cli/config.ex

```diff
@@ -7,14 +7,17 @@ defmodule CLI.Config do
 
   @doc """
   The name of the database (e.g. for RocksDB) to store loaded blocks in.
-
   ## Examples
-
       iex> CLI.Config.db_name(:ropsten)
       'db/mana-ropsten'
   """
   @spec db_name(atom()) :: nonempty_charlist()
   def db_name(chain_id) do
-    'db/mana-' ++ Atom.to_charlist(chain_id)
+    path = "#{System.cwd!()}/db/mana-#{Atom.to_string(chain_id)}"
+    dirname = Path.dirname(path)
+    :ok = File.mkdir_p(dirname)
+
+    "#{System.cwd!()}/db/mana-#{Atom.to_string(chain_id)}"
```

Comment:
Can we reuse the `path` variable from line 16 here?

Reply:
good point